Makefile

```diff
@@ -64,7 +64,7 @@ docker-push: $(addsuffix -push,$(IMAGES))
 docker-push-latest-dev: $(addsuffix -push-latest-dev,$(IMAGES))
 
 %-push-latest-dev:
-	docker push $(DOCKER_REGISTRY)/$*:$(PRESIDIO_LABEL)
+	docker pull $(DOCKER_REGISTRY)/$*:$(PRESIDIO_LABEL)
```

Comment:
Is this for pulling before tagging or for pushing both the label and the 'latest'?